deck/models.py

```diff
@@ -349,6 +350,28 @@ def get_not_approved_schedule(self):
             .order_by('-is_approved', '-votes__rate__sum')
         return not_approved_schedule
 
+    def get_range_page(self, page_number):
```

Comment:
I think that this method can be moved to `deck_tags.py`. You created a method on the Event model, that does not use the event instance and then when you need to call that method you are creating an unnecessary object just to call it.